probatus/interpret/shap_dependence.py

```diff
@@ -225,22 +233,55 @@ def plot(
             raise ValueError("alpha must be a float value betwee 0 and 1")
 
         self.min_q, self.max_q, self.alpha = min_q, max_q, alpha
+        if other_tdp:
+            other_tdp.min_q, other_tdp.max_q, other_tdp.alpha = min_q, max_q, alpha
```

Comment:
What if the other_tdp has lower min_q or higher max_q?
Maybe good to take min and max over quantiles in both plots